### DISCUSSION TITLE
Where are new programme files saved?

### DISCUSSION THREAD

**Discussion by jeagh** [pybricks/support#920] [Q&A]
Another question and I promise to keep them to a minimum.

Just out of curiosity where are new programme files saved? I'm not referring to the saving of backup files.

I have used three different operating systems and four different browsers. Each browser only displays those files that were created in it and so I'm wondering where those files are saved. It seems not in the operating system's file system.

**Comment by dlech** [comment]
They are saved using the indexeddb feature of the web browser. Since the "files" are in a database, there are no actual files on your computer and why each browser shows different files.

**Reply by jeagh** [reply]
Thank you again dlech. I'll investigate the indexdedb browser feature.

**Comment by laurensvalk** [comment]
Reading between the lines, I think you might be asking about sharing files conveniently between computers and/or users.

That's not something we have planned for the moment. See also https://github.com/orgs/pybricks/discussions/811

**Reply by jeagh** [reply]
Thank you for the extra information. Sharing files wasn't on my mind when I asked the question, but now that you mention it that would be a worthy convenience. However, from what I've read about indexeddb it might be an unnecessary complication.
